skimage/filters/thresholding.py

```diff
@@ -912,7 +929,9 @@ def threshold_sauvola(image, window_size=15, k=0.2, r=None):
     image: (N, M) ndarray
```

Comment:
the image can be n-d I think, at least 3D, therefore the `(N, M)` shape should be removed.